crates/egui/src/containers/window.rs

```diff
@@ -405,10 +405,19 @@ impl<'open> Window<'open> {
         let resize = resize.resizable(false); // We move it manually
         let mut resize = resize.id(resize_id);
 
+        let on_top = Some(area_layer_id) == ctx.top_layer_id();
         let mut area = area.begin(ctx);
 
         let title_content_spacing = 2.0 * ctx.style().spacing.item_spacing.y;
 
+        // Calculate roughly how much larger the window size is compared to the inner rect
+        let title_bar_height = if with_title_bar {
+            let style = ctx.style();
+            ctx.fonts(|f| title.font_height(f, &style)) + title_content_spacing * 2.0
```

Comment:
This (the `* 2.0`) is the culprit. When set to `1.0`, the result is correct:


![image](https://github.com/emilk/egui/assets/30444886/b6a00d70-4e29-47a1-afc4-feec0900326d)

Reply:
Good catch - I'll revert it

Reply:
I'm curious why only your window is broken, and no other one on egui.rs

Maybe you can make a PR to fix it?